packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -59,6 +59,7 @@ export const contentClass = style([
     display: 'flex',
     flexDirection: 'column',
     gap: '$xs',
+    width: '100%',
```

Comment:
fixes an issue with the size of the Icon. for some reason with a long text, the icon gets super small

![image](https://github.com/kadena-community/kadena.js/assets/4015521/77d519cd-2db6-440e-aa14-e038d645c88b)